### DISCUSSION TITLE
Just want to say thanks

### DISCUSSION THREAD

**Discussion by byzod** [Nain57/Smart-AutoClicker#365] [General]
I still need the offset thing (#167) but after so many updates it's already good enough for my need

It's fast and reliable, more than any mobile clicker I've seen, feels even faster than `ImageSearch` of autohotkey I don't know how you did that :D

And it's open souce

Thanks dude, for keep updating this

**Comment by Nain57** [comment]
Hi and thank you for your support !

I currently working on 3.0.0 that will add a lot of new features, and an overhaul that will ease the implementation of the offset feature. 

This app started has a hobby during the COVID, and i didn't expect to release it to the public at that time, so it has some technical debt that prevents me to release some features easily. But i'm working on it !

**Reply by byzod** [reply]
Good to hear that, good job!
